app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+# require './config/environment'
+
+class UsersController < ApplicationController
+
+  get '/users/new' do
+    erb :"users/new.html"
+  end
+
+  post '/users' do
+
+    # describe "with mismatched passwords" do
+    #   let(:user){ User.new(email: "marilyn@example.com", password: "I am a password", password_confirmation: "So am I!") }
+    #   before { user.save }
+
+    #   it { user.should_not be_valid }
+    #   it { user.errors[:password_confirmation].should include "doesn't match Password" }
+    #   # Hint: https://guides.rubyonrails.org/active_record_validations.html#confirmation
+    # end
+
+    # describe "with a non-email" do
+    #   let(:user){ User.new(email: "foo@    ") }
+    #   before { user.save }
+
+    #   it { user.should_not be_valid }
+    #   it { user.errors[:email].should include "is not a valid email address" }
+    #   # Hint: `validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: "is not a valid email address" }`
+    # end
+
+    #if the password == confirmed password
+    if params[:password] == params[:confirm]
```

Comment:
Instead of checking that the params are identical here, you should let the `User` model handle validating this.